src/components/CookieConsentBanner.js

```diff
@@ -0,0 +1,185 @@
+import React from 'react'
+import ReactGA from 'react-ga'
+import Cookies from 'js-cookie'
+
+import {
+  c_PRIMARY_TEXT,
+  c_SECONDARY_TEXT,
+  breakpoint,
+  c_COOKIE_BACKGROUND,
+  c_COOKIE_BUTTON,
+  c_COOKIE_BUTTON_HOVER,
+  GUTTER_PX,
+} from '../theme'
+import Text from './Text'
+import VerticalSpacing from './VerticalSpacing'
+import Heading from './Heading'
+
+const Button = ({ onClick, className, children, open }) => (
+  <button type="button" onClick={onClick} className={className}>
+    {children}
+  </button>
+)
+
+const ACCEPTANCE_COOKIE = 'neontribe-cookies-accept'
+const NEONTRIBE_GA = 'UA-000000-01'
```

Comment:
Have we set up a project for this yet? I think @harryharrold has admin type access to the Neontribe account, not sure who else besides me.

Reply:
The tracking ID which most @neontribe.co.uk peoples have been added to is `UA-30970110-1`

Reply:
Ah awesome. Do you know if anyone has configured it for this site at all? I've totally lost track. Maybe @roseb04 was involved?

Reply:
@zii-67 If you have time before you clock off... you could add the tracking number in.

Reply:
We had started configuring a tag manager instance. I don't think we'll need to do anything for the ga code to work, we may want to filter the view as we go.